mapiproxy/servers/default/emsmdb/emsmdbp_object.c

```diff
@@ -874,26 +875,34 @@ static int emsmdbp_copy_properties(struct emsmdbp_context *emsmdbp_ctx, struct e
 
 /* FIXME: this function is already present in oxcmsg... */
 struct emsmdbp_prop_index {
-	uint32_t display_name; /* PR_DISPLAY_NAME_UNICODE or PR_7BIT_DISPLAY_NAME_UNICODE or PR_RECIPIENT_DISPLAY_NAME_UNICODE */
-	uint32_t email_address; /* PR_EMAIL_ADDRESS_UNICODE or PR_SMTP_ADDRESS_UNICODE */
+	int64_t display_name; /* PR_DISPLAY_NAME_UNICODE or PR_7BIT_DISPLAY_NAME_UNICODE or PR_RECIPIENT_DISPLAY_NAME_UNICODE */
```

Comment:
Why don't you simply use int instead of int64_t? The code used during assignation moreover relies on a uint32_t at the moment see LOC884